ui/src/dashboards/containers/DashboardPage.tsx

```diff
@@ -421,15 +483,29 @@ class DashboardPage extends Component<Props, State> {
   private handleSaveEditedCell = async (
     newCell: DashboardsModels.Cell | NewDefaultCell
   ): Promise<void> => {
-    const {dashboard, handleHideCellEditorOverlay} = this.props
+    const {dashboard} = this.props
 
     if (this.isExistingCell(newCell)) {
       await this.props.updateDashboardCell(dashboard, newCell)
     } else if (!this.isExistingCell(newCell)) {
       this.props.addDashboardCellAsync(dashboard, newCell)
     }
 
-    handleHideCellEditorOverlay()
+    this.handleHideCellEditorOverlay()
+  }
+
+  private handleShowCellEditorOverlay = cell => {
```

Comment:
1. yeppers. 
2. makes sense to me

Reply:
👍 i guess when i think of an overlay in this case then, i would imagine that it would source a currently selected cell from redux state, for example, rather than having to pass the cell to render in with it

Reply:
it has to populate the cell in redux and this cell is used for that.